src/main/java/com/github/mylibrelab/ui/UIStyle.java

```diff
@@ -89,7 +91,7 @@ private UIStyle() {
     }
 
     @NotNull
-    public static <T extends JComponent> T withFontFamily(final @NotNull T component, final @NotNull String fontName) {
+    public static <T extends JComponent> T withFontFamily(@NotNull final T component, @NotNull final String fontName) {
```

Comment:
`JComponent` is a subclass of `Component`. In swing it is almost 100% guaranteed that any component inside the view hierarchy is of type `JComponent`. Though some api methods only return a `Component`. I changed the type in the map because nothing specific to `JComponent`s was used. Seems like I missed it for the font methods.

Reply:
I totally forgot about swing `Component`, I guess that's how rarely I encountered it. I assumed it's something we extended from `JComponent`, and its just not yet merged. :D